diffpy/snmf/correlation.py

```diff
@@ -1,19 +1,23 @@
 import numpy as np
 
 
-def pearson(data):
+def pearson(data1, data2):
     """
+    Returns the pearson correlation coefficient between two data PDF/XRD patterns
 
     Parameters
     ----------
-    data
-
+    data1: ndarray
+        An array of the values of a XRD or PDF pattern
+    data2: ndarray
+        An array of the values of a XRD or PDF pattern
     Returns
     -------
+    float
+        The pearson correlation coefficient between data1 and data2
 
     """
-    data = np.asarray(data)
-    x = data[:, 0]
-    y = data[:, 1]
-    correlation_matrix = np.corrcoef(x, y)
+    data1 = np.asarray(data1)
+    data2 = np.asarray(data2)
+    correlation_matrix = np.corrcoef(data1, data2)
```

Comment:
can we make the variable name clearer so the code is easier to read?  something like
```
pearson_coefficient = np.corrcoef(data1, data1)[0,1]
return pearson_coefficient
```

Reply:
I will correct that